receiver/expvarreceiver/README.md

```diff
@@ -0,0 +1,49 @@
+# Expvar Receiver
+
+An Expvar Receiver scrapes metrics from [expvar](https://pkg.go.dev/expvar), 
+which exposes data in JSON format from an HTTP endpoint. The metrics are 
+extracted from the `expvar` variable [memstats](https://pkg.go.dev/runtime#MemStats), 
+which exposes various information about the Go runtime.
+
+> :construction: This receiver is in development and incomplete. It should not be used yet.
+
+## Configuration 
+
+### Default
```

Comment:
Once we know the list of default metrics we are going to export, we should add them here in that PR with an example of how to disable them if you're not interested in `gc_pause.time` for example.